posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+<!--
+.. title: Spot the differences: what is new in Spyder 5?
+.. slug: spot-the-diffenrences
+.. date: 2021-04-16 08:00:00 UTC-06:00
+.. author: Isabela Presedo-Floyd and Juanita Gomez
+.. tags: Spyder, UX/UI, release
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+![Spyder 5 versus Spyder 4](/images/spyder5-header.png)
+
+*This blog post is a conversation. Portions lead by Juanita Gomez are marked 
+as **Juanita**, and those lead by Isabela Presedo-Floyd are marked as 
+**Isabela**.*
+
+## What did we do?*
+
+**[Juanita]** [Spyder](https://www.spyder-ide.org/) was created 10 years ago and it has had the 
+contributions of a great number of developers who have written code, proposed 
+ideas, opened issues and tested PRs in order to build a piece of Spyder on 
+their own. We (Spyder’s team) have been lucky to have such a great community 
+of people contributing, but this is the first time that we decided to ask for 
+help from an UX/UI expert! Having the help of this great amount of people has 
+resulted in inconsistencies around Spyder’s interface which we didn’t stop to 
+analyze until now. 
+
+When Isabela joined Quansight, we realized that we had an opportunity of 
+improving Spyder’s interface with her help. We thought her skill set was 
+everything we needed to make Spyder’s UI better. We conducted a community 
+survey a couple of months ago and realized that some of the most common 
+feedback from users is related to its interface (very crowded, not 
+consistent, many colors). This is the reason why we decided to start a joint 
+project with Isabela, (who we consider now part of Spyder’s team) called 
+[Spyder 5](https://github.com/spyder-ide/spyder/releases/tag/v5.0.0)!!!
+
```

Comment:
I would add something like - this blogpost is an attempt to document the long and complex process of improving Spyder's UI. To give this a nice intro.

Reply:
I like the idea of adding this but maybe at the beginning along with the mention of the release? Cause I think in between these two paragraphs it interrupts a bit the flow since the next paragraph starts with the "version" of spyder which was mentioned at the end of the paragraph before. What do you think @trallard and @isabela-pf ?

Reply:
I could either option, but maybe it makes the most sense wherever the `*This blog post is a conversation…*` goes. Both are meant to give the post context and explain the motivation, so I would keep them together.